second_task/src/Sword.ts

```diff
@@ -1 +1,20 @@
-// your code goes here
\ No newline at end of file
+import { Weapon } from './Weapon';
+
+const MAX_PERCENTAGE = 1.25;
+export class Sword extends Weapon {
+  constructor(name: string, baseDamage: number, baseDurability: number, value: number, weight:number) {
+    super(name, baseDamage, baseDurability, value, weight)
+    this.name = name;
```

Comment:
@dianalemen, you don't need to assign variable in the derived class if you already pass them to the parent.